src/main/resources/swagger.api/mod-linked-data.yaml

```diff
@@ -123,9 +135,11 @@ paths:
         '500':
           $ref: '#/components/responses/internalServerErrorResponse'
 
-  /resource/metadata/{inventoryId}/id:
+  /linked-data/inventory-instance/{inventoryId}/resource-id:
```

Comment:
It shouldn't be limited to instance resource type, there are Authority types with inventoryId also.
We have renamed InstanceMetadata class to FolioMetadata because of that.
So why don't we use just /linked-data/resource/metadata/{inventoryId}/id?

Reply:
@pkjacob what do you think?

Reply:
done